aiida_mlip/calculations/geomopt.py

```diff
@@ -81,7 +81,7 @@ def define(cls, spec: CalcJobProcessSpec) -> None:
         )
 
         spec.input(
-            "opt_kwargs",
+            "minimize_kwargs",
             valid_type=Dict,
             required=False,
             help="Other optimisation keywords",
```

Comment:
```suggestion
            help="All other GeomOpt keyword arguments",
```
Admittedly, `janus-core` has the same text, but I think it is incorrect.